Assembler.Base/Creators/RawMessageInAssemblyCreator.cs

```diff
@@ -0,0 +1,21 @@
+﻿using Assembler.Core;
+using Assembler.Core.RawAssemblingEntities;
+
+namespace Assembler.Base.Creators
+{
+    public class RawMessageInAssemblyCreator : IMessageInAssemblyCreator<RawMessageInAssembly>
```

Comment:
No tests

Reply:
It's a creator...

Reply:
So? It creates entity with values in properties

Reply:
f